src/components/pages/ItemPage.js

```diff
@@ -32,98 +37,121 @@ const ItemPage = () => {
     fetchData();
   }, [url]);
 
-  // Is this the right way to do this, or is there a better way to set the state of stock using async / await?
+  // Setting the number of items in stock
   useEffect(() => {
-    setStock(item && item[0].stockCount);
+    setNumberInStock(item && item.stockCount);
   }, [item]);
 
+  // Update the number of current items in the cart
+  useEffect(() => {
```

Comment:
I really like how use used a `useEffect` to manage this. Really good. I'd just recommend not using one char variables though - like `n` - since they can be confusing. Really the only time you should use one char variables is in a for loop.